tests/test_collections.py

```diff
@@ -24,6 +24,13 @@
 from tests.fixtures.cloud_uploader import ChecksummingSink
 from dss.util.version import datetime_to_version_format
 from dss.util import UrlBuilder
+from dss import Replica
+from dss.collections import (CollectionData,
+                             get_collection,
+                             get_collections_for_owner,
+                             put_collection,
+                             patch_collection,
+                             delete_collection)
 
 
 class TestCollections(unittest.TestCase, DSSAssertMixin, DSSUploadMixin):
```

Comment:
What testmode is this set to ? integration?

Reply:
Should be normal.

Reply:
Natan just went through all the tests and was reorganizing them; https://github.com/HumanCellAtlas/data-store/blob/master/tests/readme.md 
 because some of these hit the Dynamodb/lambda should they be identified as integration?

Reply:
Good point, will change.